pkg/store/json2Values.go

```diff
@@ -0,0 +1,74 @@
+// Copyright 2019-present Open Networking Foundation.
```

Comment:
I know I might be old fashioned, but I'd go for `jsonToValues.go`

Reply:
done